expression/bench_test.go

```diff
@@ -839,6 +840,10 @@ func testVectorizedBuiltinFunc(c *C, vecExprCases vecExprBenchCases) {
 	for funcName, testCases := range vecExprCases {
 		for _, testCase := range testCases {
 			ctx := mock.NewContext()
+			if funcName == ast.AesDecrypt {
+				err := ctx.GetSessionVars().SetSystemVar(variable.BlockEncryptionMode, "aes-128-ecb")
+				c.Assert(err, IsNil)
+			}
```

Comment:
Why we need this？

Reply:
if not add this, we will have a error at here https://github.com/pingcap/tidb/blob/559f5ea87cfd3531a528c29778168b9b9292ef46/expression/builtin_encryption.go#L123